pkg/k8s/wrapper.go

```diff
@@ -75,20 +52,6 @@ var (
 		[]string{"resource_name"},
 	)
 
-	getPodFromAPIServeCallCount = prometheus.NewCounter(
-		prometheus.CounterOpts{
-			Name: "get_pod_from_api_server_call_count",
-			Help: "The number of requests to get the pod directly from API Server",
-		},
-	)
-
-	getPodFromAPIServeErrCount = prometheus.NewCounter(
-		prometheus.CounterOpts{
-			Name: "get_pod_from_api_server_err_count",
-			Help: "The number of requests that failed to get the pod directly from API Server",
-		},
-	)
-
 	broadcastPodEventCallCount = prometheus.NewCounter(
```

Comment:
This seems unused local variables. Will `go lint` catch these? Might be a good idea to run it find other unused local variables.

Reply:
[golint](https://github.com/golang/lint) seems to have been deprecated, found one more tool [go-tools](https://github.com/dominikh/go-tools) which do the job.